src/Raven.Server/Web/System/OngoingTasksHandler.cs

```diff
@@ -163,7 +167,175 @@ private IEnumerable<OngoingTask> CollectExternalReplicationTasks(List<ExternalRe
             
             return taskInfo;
         }
-        
+
+        [RavenAction("/databases/*/admin/periodic-backup/test-credentials", "POST", AuthorizationStatus.DatabaseAdmin)]
+        public async Task TestPeriodicBackupCredentials()
+        {
+            // here we explictily don't care what db I'm an admin of, since it is just a test endpoint
```

Comment:
This is already checked, in this db you are the admin.

Reply:
Didn't understand, you saying that the comment is not right here because we are an admin?

Reply:
The comment isn't right, because the router already verified that we are admin of the particular database that we are using, yes.